buildSrc/src/main/kotlin/org/cqfn/save/buildutils/DockerStackConfiguration.kt

```diff
@@ -25,6 +25,7 @@ fun Project.createStackDeployTask() {
     }
 
     tasks.register<Exec>("deployDockerStack") {
+        dependsOn("update")
```

Comment:
In [docs](https://github.com/liquibase/liquibase-gradle-plugin#usage) they say that you can use `liquibaseTaskPrefix` to customize task names, let's convert it to `liquibaseUpdate`. It's a very confusing name now.

Reply:
Fixed